_oasis

```diff
@@ -56,7 +56,7 @@ Library xenopsd
     Xenops_server_plugin,
     Xenops_server_skeleton,
     Scheduler,
-    Path,
+    Xpath,
```

Comment:
Actually, now I think `Dom0_resources` would be a more descriptive name.

Wouldn't renaming the module be a breaking change though, as I think `xenopsd` is exposed as a findlib library?